source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,175 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released recently! Given our commitment to
+[semantic versioning](http://semver.org/), this update come with 
+breaking changes for people that are using Rails below 4.2. If we 
+did introduce any bugs, please let us know, and we'll get 
+a patch release out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes 258 commits and 56
+merged pull requests from 27 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+RSpec Rails was release with full support and many improvements for 
+Rails 6.
+
+### Breaking changes
+
+This version no longer support Ruby bellow 2.3. We soft support
+Rails 4.2 but we encourage you to use rspec-rails with Rails 5
+at least.
+
+### JRuby support
+
+We very happy to support JRuby on this new release. You can now use
+`rspec-rails` with Rails 5.2 and 6 and JRuby.
+
+### Support for Action Cable
+
+Using a [channel spec][1] it will be easy to test your Action Cable
+channels. A `channels spec` is a thin wrapper for an
+`ActionCable::Channel::TestCase`, and includes all of the behavior
+and assertions that it provides, in addition to RSpec's own 
+behavior and expectations.
+
+It also includes helpers from `ActionCable::Connection::TestCase`
+to make it possible to test connection behavior.
+
+### Support for Action Mailbox
+
+Rails 6 came with [Action Mailbox][2]. We can now test them using
+`:mailbox` flag.
+
+```ruby
+```
+
+TODO: We do no have feature test for this...
```

Comment:
I saw this. Do you think we should add feature tests for ActionMailbox?

Related: https://github.com/rspec/rspec-rails/pull/1806/files

Reply:
Yes, but we shouldn't have a todo here :joy:

Reply:
For me I added a ToDo because it was mandatory to release this documentation. Do you think it should not be mandatory?

Reply:
I will try to do this change tomorrow (install ActioMailbox on generated_app, write all feature tests, add basic documentation and link here)